Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -110,6 +111,28 @@ static inline bool pageIsBeingDismissed(Document& document)
     return frame && frame->loader().pageDismissalEventBeingDispatched() != FrameLoader::PageDismissalType::None;
 }
 
+// https://html.spec.whatwg.org/multipage/images.html#updating-the-image-data:list-of-available-images
+static bool canReuseFromListOfAvailableImages(const CachedResourceRequest& request, const String& crossOriginAttribute, Document& document)
```

Comment:
Maybe we can just compare `request.options()` and `resource.options()` since `request` is created based on `crossOriginAttribute`, i.e. there is no need to pass in `crossOriginAttribute`.

Reply:
It seems to be a perfect solution:)